docs/source/Appendices/RABET-V_Glossary.md

```diff
@@ -1,101 +1,79 @@
 # RABET-V Glossary
 
-### 1st Degree Component
+```{glossary}
```

Comment:
@JDziurlaj Just noting that I turned the glossary into a Sphinx glossary.

Reply:
So how does this work? Does it look for entries in this location?